maker/src/cfd.rs

```diff
@@ -185,6 +200,7 @@ pub struct Actor<O, T, W> {
     oracle: xtra::Address<O>,
     time_to_first_position: xtra::Address<time_to_first_position::Actor>,
     connected_takers: HashSet<Identity>,
+    taker_versions: HashMap<Identity, Option<TakerDaemonVersion>>,
```

Comment:
🙃 Could also just be `HashMap<Identity, TakerDaemonVersion>` and we just don't insert in case there is no version. We default to `None` (resulting in `Old`) anyway if we don't find an entry.